### ISSUE TITLE
Loaded the model in 157.82 seconds...

### ISSUE THREAD

**Issue by Username69992872** [oobabooga/text-generation-webui#840]
### Describe the bug

Just installed with the zip autoinstaller and vicuna-13b-GPTQ-4bit-128g,
start-webui gives me `Loaded the model in 157.82 seconds.` ..which looks like to be far too long. In a YT Video I saw 6 seconds.
What could be the issue?

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

just installed oobabooga-windows.zip manualy adding vicuna-13b-GPTQ-4bit-128g

### Screenshot

![cmd_YVbxMM5dpX](https://user-images.githubusercontent.com/96194225/230485012-193c93a3-08d7-452e-8431-9d3a84971796.png)
![cmd_tTaLAmPAUM](https://user-images.githubusercontent.com/96194225/230485195-2e4e48a7-f588-479c-a3c5-5caa358e4a85.png)

### Logs

```shell
see sceenshot
```

### System Info

```shell
Win10, RTX4090, Ryzen16Core, 128GB Ram
```

**Comment by BadisG** [comment]
Yeah same, I'm sure it was loading faster before, I don't know what happend

**Comment by serpuun** [comment]
lmao, it's nothing compared to my 560 seconds
